lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9043,6 +9043,74 @@ using namespace Js;
         return DefineOwnPropertyDescriptor(arr, propId, descriptor, throwOnError, scriptContext);
     }
 
+    // ES2017: 9.4.5.3
```

Comment:
Section numbers can change, but links aren't supposed to, so could you please reference the link like this? [https://tc39.github.io/ecma262/#sec-integer-indexed-exotic-objects-defineownproperty-p-desc](https://tc39.github.io/ecma262/#sec-integer-indexed-exotic-objects-defineownproperty-p-desc)